src/regtech_data_validator/phase_validations.py

```diff
@@ -504,9 +504,9 @@ def get_phase_1_and_2_validations_for_lei(context: dict[str, str] | None = None)
                     id="W2006",
                     name="credit_purpose_ff.multi_invalid_number_of_values",
                     description=(
-                        "'Credit purpose' and 'free-form text field for other credit "
-                        "purpose' combined should not contain more than three values. "
-                        "Code 977 (other), within 'credit purpose', does not count "
+                        "* 'Credit purpose' and 'free-form text field for other credit "
+                        "purpose' combined should **not** contain more than three values. "
+                        "Code 977 (other), within 'credit purpose', does **not** count "
```

Comment:
This one is still the non-`dedent` flavor.

Reply:
I only used dedent on descriptions with actual multiple lines, the \n's.  This is a single line description, but split into multiple strings for readability.